cw_someta.php

```diff
@@ -174,7 +174,10 @@ private function getCategoryInfo($category_id) {
 		$params = json_decode($category->params);
 
 		if($params->image) {
-			$category_info['image'] = JURI::base() . $params->image;
+			// Got error 'PHP message: PHP Notice:  
+			// Trying to get property 'image' of non-object in plugins/system/cw_someta/cw_someta.php
+			// $category_info['image'] = JURI::base() . $params->image;
+			$category_info['image'] = JURI::base() . $params["image"];
```

Comment:
Thanks, but I'm not sure this will work. It is inside a conditional that checks for `$params->image` first. I'll look into it and see if I can resolve it.

Reply:
For my part, more error as well, and, no new error.
I may not have tested the tool correctly, because, I admit not knowing what this code is for, I did not make the relation between this code and its operation.
On the other hand, my readings on the error left me to think that it would be the right method.

Please watch if you have time.
It is only a NOTICE but if corrected, so much the better.